src/ApiService/ApiService/onefuzzlib/NotificationOperations.cs

```diff
@@ -142,4 +143,8 @@ public NotificationOperations(ILogTracer log, IOnefuzzContext context)
         _logTracer.Error($"unable to find crash_report or no repro entry for report: {JsonSerializer.Serialize(report)}");
         return null;
     }
+
+    public async Async.Task<Notification> GetNotification(Guid notifificationId) {
```

Comment:
is this used ?

Reply:
Forgot to include the 2nd part. Added an endpoint for querying a single notification by id.

Will follow up with CLI change so we can do `onefuzz notifications get <notification_id>`